### PR TITLE
gn.json is created with the appropriate format

### PR DESCRIPTION
**Pull request by JyothiKumar03**
created gn.json file with its full form used often

Closes #

### What new changes did you make? Tick all applicable boxes
- [X] Added new abbreviation
- [ ] Fixed something in the source code
- [ ] Added a new feature
- [ ] Fixed the docs (README.md, CONTRIBTUING.md, etc)

### Description
Created a branch named jk_contribute
created gn.json in that branch with appropriate abbrevations
removed the content in parenthesis and updated the code
added inverted commas and updated as required
kindly review and merge my pr

###

### CODE REVIEW

File: public/server/db/gn.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "definition": "Good Night",
+    "alternatives": "Get Naked(used by couples or while sexting)"
```
Comment (Njong392) [inline public/server/db/gn.json:3]: I don't think the explanation in brackets is necessary haha.
```suggestion
    "alternatives": "Get Naked"
```

**Review by Njong392 — CHANGES_REQUESTED** [review]
Thank you very much for contributing to abbreve. Please take a look at the tiny suggestion I made.

**Comment by Njong392** [comment]
Hello there. Do you need help updating this pull request?

**Comment by JyothiKumar03** [comment]
Hey Njong!!
I was busy with the exams and I totally forgot about my pending pr's!! My
apologies! I have removed the explanation part and created a new pr!!
Hope its good to go!!

**Comment by Njong392** [comment]
I believe this PR was a duplicate of #240 so I am going to close it.

**Pull request closed without merging**